apps/server/src/modules/room/room.module.ts

```diff
@@ -1,11 +1,16 @@
 import { Module } from '@nestjs/common';
 import { CqrsModule } from '@nestjs/cqrs';
-import { RoomRepo } from './repo';
+import { RoleModule } from '@modules/role';
+import { GroupModule } from '@modules/group';
+import { RoomMemberModule } from '@modules/room-member';
 import { RoomService } from './domain/service';
+import { RoomRepo } from './repo';
+import { RoomRule } from './authorization/room.rule';
+import { AuthorizationModule } from '../authorization';
 
 @Module({
-	imports: [CqrsModule],
-	providers: [RoomRepo, RoomService],
+	imports: [CqrsModule, RoomMemberModule, GroupModule, RoleModule, AuthorizationModule],
```

Comment:
wasnt the whole purpose of having a seperate roomMemberModule to keep it out of the room Module? where did you run into issues, and need it in the roomModule? we should see if thats avoidable, otherwise we should think about combining them into one module, since it kind of defeats the point...

Reply:
Fixed